docs/working-with-projects/project-configuration/create-robot-accounts.md

```diff
@@ -13,33 +13,33 @@ You can create robot accounts to run automated operations. Robot accounts have t
 1. Log in to the Harbor interface with an account that has at least project administrator privileges.
 1. Go to **Projects**, select a project, and select **Robot Accounts**.
 
-    ![Robot accounts](../../../img/add-robot-account.png)
+    ![Robot accounts](../../img/add-robot-account.png)
 
 1. Click **New Robot Account**.
 1. Enter a name and an optional description for this robot account.
-1. Set expiration time for this robot account. If not set, the expiration time of system configuration will be used for this robot account.
+1. Set expiration time for this robot account, select checkbox **Never Expired** if you want to create a permanent robot account. If not set, the expiration time of system configuration will be used for this robot account. 
```

Comment:
Set expiration time for this robot account, you can also select checkbox **Never Expired** if you want to create a never expiring robot account.